shopBuddy/shopBuddy/ItemDetailViewController.m

```diff
@@ -9,25 +9,37 @@
 #import "Item.h"
 #import "UIImageView+AFNetworking.h"
 #import "APIManager.h"
+#import "ShoppingList.h"
+#import "ShoppingList+Persistent.h"
+#import "Parse/Parse.h"
 
 @interface ItemDetailViewController ()
-@property (weak, nonatomic) IBOutlet UILabel *titleLabel;
-@property (weak, nonatomic) IBOutlet UILabel *brandLabel;
-@property (weak, nonatomic) IBOutlet UIImageView *itemImage;
-@property (weak, nonatomic) IBOutlet UITextView *descriptionView;
+{
+    NSArray<ShoppingList*> *lists;
+    __weak IBOutlet UITextView *descriptionView;
+    __weak IBOutlet UILabel *titleLabel;
+    __weak IBOutlet UILabel *brandLabel;
+    __weak IBOutlet UIImageView *itemImage;
+    __weak IBOutlet UIButton *addItemToListButton;
+    
+}
 
 @end
 
 @implementation ItemDetailViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    self.descriptionView.scrollEnabled=YES;
+    descriptionView.scrollEnabled=YES;
     if (self.item != nil) {
         [self _populateView];
     } else {
         [self _callAPI];
     }
+    [ShoppingList fetchListsByUser:[PFUser currentUser] withCompletion:^(NSArray *lists, NSError *error) {
```

Comment:
I think this API call is unnecessary, I saw you are fetching the lists in the ShoppingManager VC, why not pass the lists there to the subsequent VC after navigation instead of having to do a second API call to get the same result.

Reply:
I have the AddItem view which doesn't use the lists between the ShoppingManager and ItemDetail view. Should I pass the lists to the AddItem view and then to the ItemDetail view even though I don't use it in AddItem or should I just retrieve the lists again in ItemDetail?

Reply:
You can do this in the next PR as we discussed.